### DISCUSSION TITLE
How to enable hyphenate in raw text?

### DISCUSSION THREAD

**Discussion by asmarcz** [typst/typst#3190] [Q&A] [answered]
Hello, how would I go about enabling hyphenation for raw text?

**Comment by frozolotl** [comment, marked as answer]
This should do the trick:
```typ
#show raw: set text(hyphenate: true)
```

Instead of `#show raw: …`, you might also want to use `#show raw.where(block: true): …` depending on your use-case.

**Reply by asmarcz** [reply]
Thank you! I didn't realise the raw text is still just a text.. hah.
